packages/react-instantsearch/src/core/createInstantSearchManager.test.js

```diff
@@ -24,15 +24,16 @@ const response = {
   ],
 };
 client.search = jest.fn((queries, cb) => {
+  const clonedResponse = JSON.parse(JSON.stringify(response));
```

Comment:
Example of tests updates I had to do since new helper cc @bobylito, I cannot explain it right now but without cloning the Algolia response the tests would fail like if something was reusing the response reference